omnibus/package-scripts/chef-workstation/postinst

```diff
@@ -58,7 +58,11 @@ if is_darwin; then
 
   # Restart the app if it was running.
   echo "Restarting Chef Workstation App..."
-  $INSTALLER_DIR/bin/$app_launcher load
+  echo "USER: $USER"
```

Comment:
Could we remove the echos? The output is not actionable or informative to the operator .

Reply:
Agree.

Reply:
100%!! I need to:

1) wait for the Chef Workstation App to bump its version
2) rebase this branch
3) remove the echos
4) merge!

This build is just for debugging as the [commit message says](https://github.com/chef/chef-workstation/pull/938/commits/cf969b278c776b26dcbde99acb1490423feeeef0) it. I will remove that commit in a bit. thanks for the prompt review

Reply:
@jonsmorrow @marcparadise could you review again now? I have removed the echos and rebased this branch! 💯